README.md

```diff
@@ -12,10 +12,11 @@
 npm install -g dynamodb-admin
 
 # For Windows:
-export DYNAMO_ENDPOINT=http://localhost:8000
+set DYNAMO_ENDPOINT=http://localhost:8000
 dynamodb-admin
 
 # For Mac/Linux:
+export DYNAMO_ENDPOINT=http://localhost:8000
 DYNAMO_ENDPOINT=http://localhost:8000 dynamodb-admin
```

Comment:
It's either on or the other.

`export` will persist the export for the current session while just prefixing the command will only expose it for the duration of the command.

So it doesn't really make sense to have both.

I prefer prefixing as it doesn't leak env into the current session.

Reply:
you're right - fixed